packages/cadl-typescript/src/transform/transform.ts

```diff
@@ -96,7 +100,8 @@ export function transformUrlInfo(program: Program): UrlInfo | undefined {
           urlParameters.push({
             name: key,
             type: getSchemaForType(program, type).type,
-            description: getDoc(program, type)
+            description: getDoc(program, type),
+            value: type.kind === "String" ? type.value : undefined
```

Comment:
wonder if we should consider there're other types of const values in the urlParameters. like boolean or number ?

Reply:
Yeah, that is a good point, let me see what I can do here